packages/plugins/src/TableOfContentsPlugin.ts

```diff
@@ -43,7 +43,8 @@ const TableOfContentsPlugin: PluginType<TableOfContentsPluginPage, TableOfConten
           .map(validHeading => {
             const text = parsePageHeading(page, validHeading.value);
             return {
-              level: validHeading.depth,
+              depth: validHeading.depth,
+              level: validHeading.depth - minRank,
```

Comment:
This was the simplest way to be consistent. Now the level will be relative to the minRank so by default h2 = level 0